electron/main.ts

```diff
@@ -96,9 +96,9 @@ async function createWindow(): Promise<BrowserWindow> {
     width: 1200,
     show: !(exitToTray && startInTray),
     webPreferences: {
+      nodeIntegration: true,
       webviewTag: true,
-      contextIsolation: false,
```

Comment:
Don't change these

Reply:
Yeah those 2 files weren't supposed to be in the commit, sorry about that